index.ejs

```diff
@@ -66,7 +67,7 @@
 <% } %>
 
 <% for (var chunk in htmlWebpackPlugin.files.chunks) { %>
-<script src="<%= htmlWebpackPlugin.files.chunks[chunk].entry %>"></script>
+<script src="<%=prependUrl%><%= htmlWebpackPlugin.files.chunks[chunk].entry %>"></script>
```

Comment:
DOH ! I didn't remember that one !

I'll make a test (which I'm sure will work) and I let you know ...

Thanks and apologize me !